setup.py

```diff
@@ -29,7 +31,8 @@ def get_version():
         'msgpack-python',
         'asgiref>=1.0.0',
     ],
-    extras_requires={
-        "cryptography": ['cryptography>=1.3.0'],
+    extras_require={
+        "cryptography": crypto_requires,
+        "tests": crypto_requires + ['tox', 'asgi_ipc', 'twisted', 'txredisapi'],
```

Comment:
You can specify multiple extras using `asgi_regis[cryptography,tests]` - are you splitting them up like this for another reason?